### PR TITLE
Add big-dipper setup scripts

### LINKED ISSUE #48: Deploy big dipper block explorer
It should be visible at https://bigdipper.coralnet.cosmwasm.com

### PR DESCRIPTION
**Pull request by orkunkl**
Resolves https://github.com/CosmWasm/testnets/issues/48
These scripts are for setting up big-dipper

### CODE REVIEW

File: devops/big-dipper/big-dipper.service [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+[Service]
+TimeoutStartSec=5
+Restart=always
+User=root
+Group=root
+WorkingDirectory=/root/
+ExecStartPre=/usr/bin/docker-compose pull --quiet --ignore-pull-failures
+ExecStartPre=/usr/bin/docker-compose build --pull
+ExecStart=/usr/bin/docker-compose up --remove-orphans
```
Comment (ethanfrey) [inline devops/big-dipper/big-dipper.service:14]: Never used docker-compose. Wonder where the target is set (i expect a repo reference or such)

File: devops/big-dipper/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+version: '3'
```
Comment (ethanfrey) [inline devops/big-dipper/docker-compose.yml:1]: We just copy this to `/root` and it automatically gets picked up by the service script?
And the `.env` file as well?

I'd love to see that reference explicit.  I couldn't see this in any of the scripts.

Comment (ethanfrey) [reply devops/big-dipper/docker-compose.yml:1]: Maybe nicer to copy it to a "safer" location, as "/root" may have users mucking around. Somewhere, say, `/etc/bigdipper` would not

**Review by ethanfrey — APPROVED** [review]
Looks good. And great job getting this up and running.

Seems there is some magic in docker-compose I don't know about, but if this works on your machine, I guess it is good. I'd love some references to be a bit more explicit

**Comment by orkunkl** [comment]
I had to come up with this solution. Big dipper expects configuration JSON as env variable and I had issues using `sed`. `WorkingDirectory=/root/` sets the working direction for this service. Then .env is picked up by docker-compose https://docs.docker.com/compose/environment-variables/#the-env-file

**Comment by orkunkl** [comment]
https://community.hetzner.com/tutorials/docker-compose-as-systemd-service here are is reference doc.

**Comment by ethanfrey** [comment]
Ah,okay, I see `WorkingDirectory=/etc/big-dipper/` and the copy to `/etc/big-dipper` now.

Thanks for clarifying

**Pull request merged**